nntrainer/include/layer.h

```diff
@@ -91,6 +92,7 @@ typedef enum {
   LAYER_FLATTEN,
   LAYER_LOSS,
   LAYER_ACTIVATION,
+  LAYER_ELEM_ADD,
```

Comment:
For parsing mechanism to work correctly, the order should be:

```
  LAYER_ACTIVATION,
  LAYER_ELEM_ADD,
  LAYER_LOSS
```

I made a patch for activation layer in #393

Reply:
Done.